src/utils/__tests__/contract.test.js

```diff
@@ -1,14 +1,14 @@
 import { greaterThan } from '../../helpers/utilities';
 import Contract from '../contract';
 
-const { getAllowance } = Contract;
+const { getRawAllowance } = Contract;
```

Comment:
should we start running this on CI?

Reply:
that would be a good idea - I think the other tests we will still need to update and fix first!